build.gradle

```diff
@@ -14,9 +14,9 @@ ext.mainClassName = "com.datastax.sparkstress.SparkCassandraStress"
 
 def determineConnectorVersion() {
     if (against == 'dse') {
-        def connector = fileTree(dir: "$DseResources/spark/lib", include: 'spark*connector*_*.jar')
+        def connector = fileTree(dir: "$DseResources/spark/lib", include: 'spark*connector*jar')
         def connectorJarName = (connector as List)[0].name
-        def match = connectorJarName =~ /connector.*_.*-(\d+\.\d+.\d+).*\.jar/
+        def match = connectorJarName =~ /.*-(\d+\.\d+.\d+).*jar/
```

Comment:
won't this match anything with semver naming?

Reply:
It's been working so far, I'm open to something more narrow if you have a suggestion.
Here are some of the names we've encountered so far:
```
$ find dse-6.0.0/resources/spark/lib/ -name "spark*connector*jar"
dse-6.0.0/resources/spark/lib//spark-connector-6.0.0.jar
...
$ find dse-5.1.6/resources/spark/lib/ -name "spark*connector*jar"
dse-5.1.6/resources/spark/lib//spark-cassandra-connector-unshaded_2.11-2.0.5.jar
...
$ find dse-5.0.12/resources/spark/lib/ -name "spark*connector*jar"
dse-5.0.12/resources/spark/lib//spark-cassandra-connector-unshaded_2.10-1.6.8.jar
```

Reply:
i think maybe it should just be spark-cassandra-connector.*(\d+\.\d+\.\d)?*.jar

Reply:
not a big deal though

Reply:
changed to `spark-cassandra-connector.*(\d+.\d+.\d).*jar`, the above example didn't work

Reply:
I'm not too familiar with Scala's regex literals, but I'd be surprised if `.` isn't a meta-character that needs to be escaped.

Reply:
Rocco, how about `spark.*?connector.*?(\d+\.\d+.\d+)\.jar`? I don't think you need a wildcard after the version based on your examples. Also, unless you're trying to validate the format `[\d.]+` is a simpler and more liberal way to get the version.

Reply:
```
>>> xs = ["spark-connector-6.0.0.jar", "spark-cassandra-connector-unshaded_2.11-2.0.5.jar", "spark-cassandra-connector-unshaded_2.10-1.6.8.jar"]
>>> map(lambda x: re.match("spark.*?connector.*?([\d.]+)\.jar", x).groups(), xs)
[('6.0.0',), ('2.0.5',), ('1.6.8',)]
```